b3lb/rest/endpoints/b3lb/endpoints.py

```diff
@@ -51,58 +51,111 @@
     "getDefaultConfigXML"
 ]
 
+ENDPOINTS_WITH_INTERNAL_ID = [
+    "create",
+    "join",
+    "end",
+    "isMeetingRunning",
+    "getMeetingInfo",
+    "setConfigXML"
+]
+
 LEGAL_ENDPOINTS = WHITELISTED_ENDPOINTS + BLACKLISTED_ENDPOINTS
+SUPPRESSED_PARAMS = ['dialNumber', 'voiceBridge']
```

Comment:
I wonder if this setting should be moved to settings with a sensible default. There might be use cases to allow those settings.

(Do we already have them in the Parameters model?)

Reply:
These parameters where hard before, so these aren't in the parameter model. I will expand it.